d2layouts/d2near/layout.go

```diff
@@ -94,8 +150,20 @@ func WithoutConstantNears(ctx context.Context, g *d2graph.Graph) (nears []*d2gra
 		}
 		_, isConst := d2graph.NearConstants[d2graph.Key(obj.Attributes.NearKey)[0]]
 		if isConst {
-			nears = append(nears, obj)
-			g.Objects = append(g.Objects[:i], g.Objects[i+1:]...)
+			descendantObjects, edges := pluckOutNearObjectAndEdges(g, obj)
+
+			tempGraph := d2graph.NewGraph()
+			tempGraph.Root.ChildrenArray = []*d2graph.Object{obj}
+			tempGraph.Root.Children[obj.ID] = obj
```

Comment:
lowercase to match: https://github.com/donglixiaoche/d2/blob/05a9498c992faa191b81cab437b051b0121b6752/d2graph/d2graph.go#L565

Reply:
fixed

Reply:
am i missing something on my end or did you forget a push perhaps? these aren't fixed for me

![Screen Shot 2023-04-05 at 11 05 57 AM](https://user-images.githubusercontent.com/3120367/230166001-69d693ea-9a03-425d-ba53-fc4418ecb955.png)